frontend/src/views/producer/dashboard/reservation-list/index.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import styled, { withTheme } from 'styled-components';
+import './print.css';
+import Button from 'components/button';
+import { connect } from 'utils';
+import Typography from 'components/typography';
+import { values } from 'mobx';
+import CheckBox from 'antd/lib/checkbox';
+
+const FullScreenModal = styled.article`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  display: ${({ shown }) => (shown ? 'block' : 'none')};
+  overflow: scroll;
+
+  button {
+    margin-bottom: 1rem;
+    @media print {
+      display: none;
+    }
+  }
+  & > button {
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+  }
+`;
+const ScrollContainer = styled.div`
+  margin: 3rem;
+  max-width: 1600px;
+  width: auto;
+  & > ul {
+    margin-top: 2rem;
+  }
+`;
+
+const LiStyled = styled.li`
+  list-style: none;
+  margin: 1rem;
+  display: flex;
+
+  & > div {
+    display: inline-block;
+    margin-left: 1rem;
+
+    & > ul {
+      margin: 0.5rem;
+      padding-left: 1rem;
+    }
+  }
+`;
+
+const ReservationListItem = ({ reservation, event }) => {
+  const {
+    customerType,
+    name,
+    schoolName,
+    class: className,
+    phone,
+    email,
+    paymentCompleted,
+    tickets,
+  } = reservation;
+  const isPrivateCustomer = customerType === 'private';
+  const totalCost = tickets.reduce((acc, curr) => {
+    const { price } = event.catalogById(curr.priceId);
+    return (acc += price * curr.noOfTickets);
+  }, 0);
+
+  return (
+    <LiStyled key={reservation.id}>
+      <CheckBox checked={paymentCompleted} />
+      <div>
+        {isPrivateCustomer ? (
+          <Typography
+            style={{ margin: 0, fontSize: '1.5rem' }}
+            type="largebody"
+          >
+            {name}
+          </Typography>
+        ) : (
+          <Typography
+            style={{ margin: 0, fontSize: '1.5rem' }}
+            type="largebody"
+          >
+            {name}, {schoolName}, {className}
+          </Typography>
+        )}
+        <br />
```

Comment:
How about also having a Typography for a paragraph (with p-tag) and using it here, so you wouldn't need to add `<br/>`